src/components/NavBar/index.js

```diff
@@ -133,6 +144,21 @@ class NavBar extends Component {
   }
 }
 
+const FAQIcon = ({ intl }) => (
+  <a
```

Comment:
use `window.open(`${Routes.explorer.tx}/${transaction.txid}`, '_blank');` for opening links. not sure if using a regular anchor will play nice with Electron.